x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/index.tsx

```diff
@@ -0,0 +1,259 @@
+
+const getNetworkTopNFlowColumns = (
+  startDate: number,
+  networkTopNFlowType: NetworkTopNFlowType
+): Array<Columns<NetworkTopNFlowEdges>> => [
+  {
+    name: networkTopNFlowType === NetworkTopNFlowType.source ? i18n.SOURCE_IP : i18n.DESTINATION_IP,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) => {
+      const attr =
+        networkTopNFlowType === NetworkTopNFlowType.source ? 'source.ip' : 'destination.ip';
+      const ip: string | null = get(attr, node);
+      if (ip != null) {
+        return (
+          <DraggableWrapper
+            dataProvider={{
+              and: [],
+              enabled: true,
+              id: escapeDataProviderId(`networkTopNFlow-table-${networkTopNFlowType}-ip-${ip}`),
+              name: ip,
+              excluded: false,
+              kqlQuery: '',
+              queryMatch: {
+                field: attr,
+                value: ip,
+              },
+              queryDate: {
+                from: startDate,
+                to: Date.now(),
+              },
+            }}
+            render={(dataProvider, _, snapshot) =>
+              snapshot.isDragging ? (
+                <DragEffects>
+                  <Provider dataProvider={dataProvider} />
+                </DragEffects>
+              ) : (
+                ip
+              )
+            }
+          />
+        );
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+  },
+  {
+    name: i18n.DOMAIN,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) => {
+      const attr =
+        networkTopNFlowType === NetworkTopNFlowType.source ? 'source.domain' : 'destination.domain';
+      const domain: string | null = get(attr, node);
+      if (domain != null) {
+        return (
+          <DraggableWrapper
+            dataProvider={{
+              and: [],
+              enabled: true,
+              id: escapeDataProviderId(
+                `networkTopNFlow-table-${networkTopNFlowType}-domain-${domain}`
+              ),
+              name: domain,
+              excluded: false,
+              kqlQuery: '',
+              queryMatch: {
+                field: attr,
+                value: domain,
+              },
+              queryDate: {
+                from: startDate,
+                to: Date.now(),
+              },
+            }}
+            render={(dataProvider, _, snapshot) =>
+              snapshot.isDragging ? (
+                <DragEffects>
+                  <Provider dataProvider={dataProvider} />
+                </DragEffects>
+              ) : (
+                domain
+              )
+            }
+          />
+        );
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+  },
+  {
+    name: i18n.BYTES,
+    truncateText: false,
+    hideForMobile: false,
+    render: ({ node }) =>
+      has('network.bytes', node)
+        ? numeral(node.network!.bytes).format('0.000b')
```

Comment:
I am guessing this is part of the `i18n` effort. If so, looks good to know on how to format numeral in Kibana 👍